### DISCUSSION TITLE
In Elsa 3.1.3, the Switch (flow) activity did not set the Expression property of variable parameters, resulting in the inability to pass parameters to the Switch?

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5698] [General]
In Elsa 3.1.3, the Switch (flow) activity did not set the Expression property of variable parameters, resulting in the inability to pass parameters to the Switch?
![switch](https://github.com/elsa-workflows/elsa-core/assets/171897414/24a939f1-5724-44ef-8db5-c0144b641cdb)

**Comment by glime-ai** [comment]
Based on the context provided, it seems that there might be an issue with the Switch activity in Elsa 3.1.3 where the Expression property of variable parameters is not being set correctly, which could lead to problems with passing parameters to the Switch activity. This could be a bug or a limitation in the current version of Elsa.

However, without specific error messages or code examples, it's difficult to provide a definitive answer or solution. If you are experiencing this issue, I would recommend checking the Elsa GitHub issues page for any reported bugs related to the Switch activity and looking for any updates or workarounds provided by the Elsa community. Additionally, you could try reaching out to the community or maintainers for support.

If you can provide more details or error messages related to the problem, I may be able to offer more targeted assistance.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
